.github/workflows/build-test-windows.yml

```diff
@@ -51,3 +51,36 @@ jobs:
 
       - name: Run build
         run: make build
+
+  windows-lateest-unittest:
```

Comment:
Any reason these are two separate jobs instead of reusing the same job but defining a matrix for the runner OS?

Reply:
Also there's a typo in the job name